addons/dialog/functions/fnc_gui_slider.sqf

```diff
@@ -22,11 +22,36 @@
  */
 
 params ["_controlsGroup", "_defaultValue", "_settings"];
-_settings params ["_min", "_max", "_formatting", "_isPercentage"];
+_settings params ["_min", "_max", "_formatting", "_isPercentage", "_drawRadius", "_radiusCenter", "_radiusColor"];
 
 private _ctrlSlider = _controlsGroup controlsGroupCtrl IDC_ROW_SLIDER;
 private _ctrlEdit = _controlsGroup controlsGroupCtrl IDC_ROW_EDIT;
 
+if (_drawRadius) then {
+    [missionNamespace, "Draw3D", {
+        _thisArgs params ["_ctrlSlider", "_center", "_color"];
+
+        if (isNull _ctrlSlider || {_center isEqualTo objNull}) exitWith {
+            removeMissionEventHandler [_thisType, _thisID];
+        };
+
+        if (_center isEqualType objNull) then {
+            _center = ASLToAGL getPosASL _center;
```

Comment:
Probably want to use `getPosASLVisual` here for smoother drawing around moving objects.